warehouse/packaging/interfaces.py

```diff
@@ -96,3 +89,41 @@ def create_project(name, creator, request, *, creator_is_owner=True):
         If `creator_is_owner`, a `Role` is also added to the project
         marking `creator` as a project owner.
         """
+
+
+class ProjectNameUnavailableError(Exception):
+    """Base exception for project name unavailability errors."""
+
+    pass
+
+
+class ProjectNameUnavailableInvalid(ProjectNameUnavailableError):
```

Comment:
I don't know how strict we are with PEP 8, as none of our linters yelled about this, but I had learned that [exception classes should end in `*Error`](https://peps.python.org/pep-0008/#exception-names).
We do that in other places like [these exceptions](https://github.com/pypi/warehouse/blob/0e7e85f11c84c52d037915b2880cd442ca7006ed/warehouse/captcha/hcaptcha.py#L26-L43).

I don't think it's a huge deal, but would be curious what others thought about that.

Reply:
Fixed